js/navigation.js

```diff
@@ -0,0 +1,38 @@
+// Sections
+const list = document.querySelector('#List');
+const add = document.querySelector('#New');
+const contact = document.querySelector('#Contact');
+
+// Navigation links array
+const links = document.querySelectorAll('#myLinks a');
+
+// Function for showing the chosen section when a link gets clicked
+function GoToSection(e) {
+  // Name of the section to show
+  const name = e.target.textContent;
+
+  switch (name) {
+    case 'List':
+      list.classList.remove('hidden');
+      add.classList.add('hidden');
+      contact.classList.add('hidden');
+      break;
+    case 'Add New':
+      list.classList.add('hidden');
+      add.classList.remove('hidden');
+      contact.classList.add('hidden');
+      break;
+    case 'Contact':
+      list.classList.add('hidden');
+      add.classList.add('hidden');
+      contact.classList.remove('hidden');
+      break;
+    default:
+      break;
+  }
+}
+
+// Click event listener added for each navigation link
+links.forEach((section) => {
+  section.addEventListener('click', GoToSection);
+});
```

Comment:
Please note that on page load, all the navigation details are displayed on a single page which should not be. Kindly fix it. 👍🏽
![Screenshot from 2023-07-12 22-46-39](https://github.com/alexansaa/LibraryApp/assets/44624138/e0bc2224-afd1-4297-9d63-a81cdbf6aad3)

Reply:
At the moment, I can add a `book` without `author` and vice-versa. Please ensure that all necessary parameters are included before a book can be added. 😉

Reply:
Well resolved ✔️ 

![image](https://github.com/alexansaa/LibraryApp/assets/38088818/15a5960c-9e8d-4b40-8b7a-6e408e76ef78)

Validation 👍🏾 

![image](https://github.com/alexansaa/LibraryApp/assets/38088818/82c11e1b-667a-4f68-ae66-3fad1aa43685)